apps/src/aiDifferentiation/AiDiffPredefinedPrompts.tsx

```diff
@@ -206,6 +206,125 @@ export const IMPROVE_THIS_CODE = {
   prompt: 'How can this student code be improved?',
 };
 
+export const INNOVATOR_NEW_WAYS_PROMPT = {
+  label: 'New ways to teach this lesson',
+  prompt: `I want to explore, extend, and find the next exciting thing. I’m looking for a new way to teach this lesson’s key concepts. Suggest a hands-on activity that my students could experiment with.`,
+};
+
+export const INNOVATOR_CREATIVE_EXTENSION_PROMPT = {
+  label: 'Creative extension activities for a challenge',
+  prompt: `I want to explore, extend, and find the next exciting thing. My students finished the lesson. What are some ideas or extensions to creatively challenge them?`,
+};
+
+export const CODE_WHISPERER_GUIDING_QUESTIONS_PROMPT = {
+  label: 'Guiding questions',
```

Comment:
I am wondering if this could benefit from a verb.  Like "Generate guiding questions"